helm_lib/templates/_csi_node.tpl

```diff
@@ -23,12 +23,8 @@ memory: 25Mi
   {{- $additionalNodeVolumeMounts := $config.additionalNodeVolumeMounts }}
 
   {{- $kubernetesSemVer := semver $context.Values.global.discovery.kubernetesVersion }}
-
   {{- $driverRegistrarImageName := join "" (list "csiNodeDriverRegistrar" $kubernetesSemVer.Major $kubernetesSemVer.Minor) }}
-  {{- $driverRegistrarImageTag := index $context.Values.global.modulesImages.tags.common $driverRegistrarImageName }}
-  {{- $driverRegistrarImage := include "helm_lib_module_common_image" (list $context $driverRegistrarImageName) }}
-
-  {{- if $driverRegistrarImageTag }}
+  {{- if $driverRegistrarImage := include "helm_lib_module_common_image_no_fail" (list $context $driverRegistrarImageName) }}
```

Comment:
if exists

Reply:
> fakewerf -> werf

done

> images_tags generator -> CI

internally decided to leave the generator

Reply:
I was forced to revert function `helm_lib_module_common_image_no_fail`, because  netsting `helm_lib_module_common_image` after `helm_lib_module_common_image_exists` is not working well for some reason.